.storybook/config.js

```diff
@@ -0,0 +1,112 @@
+import React from "react"
+
+import styled from "styled-components"
+import { LocationProvider } from "@reach/router"
+import { addDecorator, configure } from "@storybook/react"
+import { GlobalStyle, ThemeProvider, themeSpacing } from "@datapunt/asc-ui"
+import { addParameters } from "@storybook/react"
+
+const Wrap = styled.div`
+  padding: ${ themeSpacing(5) };
+`
+
+// automatically import all files ending in *.stories.js
+const req = require.context("../src", true, /\.stories\.(tsx)$/)
+
+const extendedTheme = {
+  globalStyle: `
+  `,
```

Comment:
Can this be an empty string `""`. Or do we need the linebreak somehow?

Reply:
I'll change it